_timeline/2018-05-30-librerouter-radio-and-antenna-test.md

```diff
@@ -0,0 +1,5 @@
+---
+title: LibreRouter Radio and Antenna Test
+date: 2018-05-30
+---
+We ran a series of tests with the LibreRouter's radio and antennas to determine how well they worked compared to off-the-shelf products. [Our Results](https://github.com/tomeshnet/documents/blob/master/technical/20180530_hpm5g-radio-tests.md)
```

Comment:
Could the last sentence be a complete sentence with a bit that links to the results?

Reply:
![image](https://user-images.githubusercontent.com/29005789/43374956-3c1ff2a4-9380-11e8-9245-ca993ebd363e.png)

Mirrored it on this one so we should change bots.

maybe

We recorded our results.

???